napari/utils/colormaps/colormaps.py

```diff
@@ -360,16 +353,15 @@ def increment_unnamed_colormap(
     return name
 
 
-def ensure_colormap_tuple(colormap: ValidColormapArg) -> Tuple[str, Colormap]:
+def ensure_colormap(colormap: ValidColormapArg) -> Tuple[str, Colormap]:
```

Comment:
```suggestion
def ensure_colormap(colormap: ValidColormapArg) -> Colormap:
```

It looks like just the `Colormap` object is returned now. If so, I think the `Returns` section of the docstring needs to be updated as well.

Reply:
Done